python/ct/crypto/ecdsa_keys.py

```diff
@@ -0,0 +1,76 @@
+from ct.crypto import error
+from ct.crypto import pem
+from ct.proto import client_pb2
+
+import hashlib
+import ecdsa
+
+class EcdsaPublicKey(object):
```

Comment:
Nit: Naming: That's more than a public key representation, that can also verify signatures. Name that accordingly? (EcdsaVerifier, EcdsaValidator, etc)